src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -62,10 +62,6 @@ public class TestConnectionCommand : PSCmdlet, IDisposable
 
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-windows`
WinRM based remoting session abrupt disconnect.Verifies that an abruptly disconnected Invoke-Command session produces a valid disconnected job needed for reconnect
```powershell
Expected exactly 'Disconnected', but got Disconnecting.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Remoting\RemoteSession.Disconnect.Tests.ps1: line 75
75:         $session.State | Should -BeExactly 'Disconnected'
```